packages/cockpit/ui/src/utils/presentation.js

```diff
@@ -2,7 +2,8 @@ import isToday from 'date-fns/isToday';
 import formatDistanceToNow from 'date-fns/formatDistanceToNow';
 
 export function formatContractForDisplay(contract) {
-  let address = (contract.deployedAddress || contract.deployedAddress);
```

Comment:
Wow, this condition was very useful.
I'm guessing the wanted line was `(contract.deployedAddress || contract.address);`

Reply:
That's what it used to be, according to git blame; it was changed in a big refactor PR, but based on the other changes in that PR it looked to me like it's okay to just check `contract.deployedAddress` rather than reintroduce `|| contract.address`.